packages/dds/shared-object-base/src/test/attachingBindingAndConnecting.spec.ts

```diff
@@ -450,16 +455,17 @@ describe("SharedObject attaching binding and connecting", () => {
 					connected && sharedObject.isAttached(),
 					"connected",
 				);
-			}),
-		);
+			});
+		}
 
 		it("isAttached with detached transition to attach runtime", async () => {
 			const runtimeEvents = new TypedEventEmitter<IFluidDataStoreRuntimeEvents>();
+
 			let didAttach = 0;
 			let attachCalled = false;
 			const { overrides, sharedObject } = createTestSharedObject({
 				runtime: {
-					...(runtimeEvents as any),
+					// ...runtimeEvents,
```

Comment:
Is this not needed? Should we remove the line?

Reply:
Oops, I think I still needed to fix something here, looking into it.